dspace-api/src/test/java/org/dspace/content/RelationshipMetadataServiceTest.java

```diff
@@ -29,17 +33,13 @@
 import org.dspace.content.service.RelationshipService;
 import org.dspace.content.service.RelationshipTypeService;
 import org.dspace.content.service.WorkspaceItemService;
-import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 
-public class RelationshipMetadataServiceTest extends AbstractUnitTest {
-
-    private static final Logger log = org.apache.logging.log4j.LogManager
-        .getLogger(RelationshipMetadataServiceTest.class);
+public class RelationshipMetadataServiceTest extends AbstractIntegrationTestWithDatabase {
```

Comment:
This file should now be renamed `RelationshipMetadataServiceIT` since it's been switched to an Integration Test instead of a Unit Test.